stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -229,3 +290,8 @@ private fun ClickableText(
         }
     )
 }
+
+/**
+ * The max length of quote message. After that it gets ellipsized.
+ */
+private const val DefaultQuoteMaxLength: Int = 170
```

Comment:
Instead of having a number of characters, maybe we can go for a line limit? as you can have 170 new lines and this will work.

I'd suggest making the limit of lines to two or three (as anything more than that is ugly) and making this constant that limit.

Reply:
I can refactor it to have ma lines. Was following the designs on character limit so we should have the same logic as on other platforms.